### ISSUE TITLE
Adding SDK examples for reference?

### ISSUE THREAD

**Issue by deividAlfa** [IOsetting/hk32f030m-template#8]
There're few SDK examples available [here](https://bbs.21ic.com/icview-3236132-1-1.html), I uploaded them [here](https://www.eevblog.com/forum/microcontrollers/$0-25-hk32f030m-(cortex-m0-32mhz-16kb-2kb)/msg4869398/#msg4869398).

They might be prety useful as reference, it woud be nice to add them to the repo files.

**Comment by IOsetting** [comment]
Yes these examples are very helpful. Because they are Keil MDK projects, they need to be tested on GCC before being added to the repository, I might do it later.

**Comment by deividAlfa** [comment]
They're pretty generic examples, being Keil projects is not important, it's using the same peripheral library, so the calls are the same.
It made my life a lot easier when using sleep/stop modes, AWU timer...

Not really meant to be "compile and run", but provide some useful info, which is kinda scarce with these obscure devices.

You know, have a peek at main.c, etc,
"Ahh it's using this function" 🤔
